LiskHQ/lisk-mobile
# Splash screen hide before the animation starts - Closes #306

# What was the bug or feature?
It is described #306

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

### How to test it?
run the app on the android.
splash screen should be visible until the time the welcome animation is ready to start.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
